fe/src/app/jobs/[jobId]/page.tsx

```diff
@@ -0,0 +1,102 @@
+import Carousel from "@/components/Carousel/Carousel";
+import Timer from "@/components/Timer/Timer";
+import { Button, Chip, Typography } from "@/lib/materialTailwindExports";
+import { AdvertType } from "@/types/AdvertType";
+import { getToken } from "@/utils/getToken";
+import { BASE_API_URL, SERVICE_ADVERTRS_ROUTE } from "@/utils/urls";
+import { FaCheckCircle } from "react-icons/fa";
+import { FaCircleXmark } from "react-icons/fa6";
+
+const AnswerIcon = (isPositive: boolean) => isPositive ? <FaCheckCircle className="text-green-500 text-2xl" /> : <FaCircleXmark className="text-red-500 text-2xl" />;
+
+async function getAdvert(jobId: string) {
+  const token = await getToken();
+  const response = await fetch(`${BASE_API_URL}${SERVICE_ADVERTRS_ROUTE}/${jobId}`, {
+    headers: { Authorization: `Bearer ${token}` }
+  });
+  if (!response.ok) {
+    return;
```

Comment:
```suggestion
    return;
    toast.error(SOMETHING_WENT_WRONG_MESSAGE)
```

Reply:
Not implemented as it is server component and then I get differences in client and server error.